_layouts/story.html

```diff
@@ -3,44 +3,52 @@
 ---
 <div class="post">
 
-  <header class="post-header">
-    <h1 class="post-title">{{ page.title }}</h1>
-  </header>
+  {% if page.wip %}
+  <div class="notice wip">
+    This is a work in progress and may need additional edits to meet the protocols.
```

Comment:
**This story is still a work in progress.** If you would like to help write it or know some useful resources that should be referenced, please share some thoughts in the following GitHub issue!